### ISSUE TITLE
[Accessibility]A11y_Java Tooling for IntelliJ/Eclipse_Eclipse Marketplace_Screen Reader: Narrator and NVDA are not reading the label for search-box

### ISSUE THREAD

**Issue by soumyatippapuram** [microsoft/azure-tools-for-java#3367]
[Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier!

GitHubTags: #A11yTCS; #A11yMAS; #DesktopApp; #Win10; #A11y_Java Tooling for IntelliJ/Eclipse-Stay Healthy; #MAS1.3.1; #Screen Reader; #C+E-H; #Java tooling for IntelliJ/Eclipse; #Narrator; #NVDA 

**Environment Details:**
Eclipse IDE for Enterprise Java Developers
Version: 2019-06 (4.12.0)
Build id: 20190614-1200

**OS Details:** 
OS: Windows 10 (Version 1903, Build: 18362.175)
MAS Violated: MAS 1.3.1

**Repro Steps: ​**
1) Start screen reader and open Eclipse application
2) Tab till Help and select "Eclipse Marketplace" from the list
3) Search/Find for 'Azure Toolkit for Eclipse' and press enter
4) Tab till the search box and check if the screen reader is reading the label for it

**Actual Result: ​**
Screen reader is not reading the label "Find" for the search box. It is just reading it as "edit"

**Expected Result: ​**
Screen reader should read it as "Find - Edit"

**User Impact: ​**
Screen reader users will not know what to enter in the text-box if it just reads as edit
 
**MAS Reference:**
[MAS 1.3.1](https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4})

[Bug_3367_Eclipse_Marketplace_Find_Narrator.pptx](https://github.com/microsoft/azure-tools-for-java/files/3451860/Bug_3367_Eclipse_Marketplace_Find_Narrator.pptx)
![Bug_3367_Eclipse_Marketplace_Find_NVDA](https://user-images.githubusercontent.com/41108985/62212760-19885000-b3bf-11e9-9fcc-f837b9576648.png)

**Comment by Flanker32** [comment]
@soumyatippapuram This is an external issue which has been linked to [AAT](https://cela-ra-aat-prod.azurewebsites.net/#/workflow/56452990-0d38-45bc-b475-a62832ce0076), please help us verify it, thanks!

**Comment by soumyatippapuram** [comment]
Reported a new issue https://bugs.eclipse.org/bugs/show_bug.cgi?id=551108 to Eclipse

**Comment by Flanker32** [comment]
Discussed this bug with Ryan and Sarah and approved to close this issue as its external and there is non blocking scenario.

**Comment by vscjavaci** [comment]
Track internal issue by AB#1749766
